HOW_TO_TEST.md

```diff
@@ -1,97 +1,115 @@
 # How to test
 
-Starting with CP-Ansible 5.5.0, we have included testing via the [Molecule](https://molecule.readthedocs.io/en/latest/) framework, and strongly advise its usage before submitting a Pull Request.
+CP-Ansible's tests use the [Molecule](https://molecule.readthedocs.io/en/latest/) framework, and it is strongly advised to test this way before submitting a Pull Request.
 
 ## Prerequisites
 
-1. Python3 installed with PIP
-2. Docker installed
-
-Note: We recommend increasing your docker memory to at least 20GB of RAM and your CPU count to 10.  
-
-### Installing Molecule via pip
-
-To install the Molecule and Docker libraries  
-
-```pip install molecule docker```
-
-Note: If using Molecule version 3.1.0 or later use the following command to install:
-
-```pip3 install molecule molecule-docker```
-
-
-### Running Molecule in a Container
-
-At times, pip installation of molecule can lead to errors:
+1. Python3
+2. [Ansible >= 2.11 installed with Pip](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-ansible-with-pip). Use python3's pip for ansible installation to make sure ansible is not configured with Python2.7:
 ```
-ImportError: No module named docker.common
+python3 -m pip install --user ansible
 ```
-
-As a workaround you can use molecule in a container.  
-In your current shell create an alias to start molecule in a container:
-
+2. [Docker](https://docs.docker.com/get-docker/) *Note: We recommend increasing your docker memory to at least 20GB of RAM and your CPU count to 10.*
+3. [Molecule](https://molecule.readthedocs.io/en/latest/installation.html#install). Use python3's pip for installation ie 
 ```
-git clone https://github.com/confluentinc/cp-ansible
-cd cp-ansible
-export CP_ANSIBLE_PATH=$PWD
-alias molecule="docker run -it --rm --dns="8.8.8.8" -v "/var/run/docker.sock:/var/run/docker.sock" -v ~/.cache:/root/.cache -v "$CP_ANSIBLE_PATH:$CP_ANSIBLE_PATH" -w "$CP_ANSIBLE_PATH/roles/confluent.test" quay.io/ansible/molecule:3.1.5 molecule"
+python3 -m pip install --user "molecule[docker,lint]"
 ```
 
-To make the alias permanent, add the following to your bashrc file:
+## Cloning CP-Ansible
+
+CP-Ansible is written as an [Ansible Collection](https://docs.ansible.com/ansible/latest/dev_guide/developing_collections.html), which requires a rigid directory structure for development. In order for Molecule to succeed in finding custom filters and plugins, cp-ansible must be cloned into an `ansible_collections/` directory. If you already have cp-ansible cloned run:
 
 ```
-export CP_ANSIBLE_PATH=<Replace this with the repo path>
-alias molecule="docker run -it --rm --dns="8.8.8.8" -v "/var/run/docker.sock:/var/run/docker.sock" -v ~/.cache:/root/.cache -v "$CP_ANSIBLE_PATH:$CP_ANSIBLE_PATH" -w "$CP_ANSIBLE_PATH/roles/confluent.test" quay.io/ansible/molecule:3.1.5 molecule"
+mkdir -p ansible_collections/confluent
```

Comment:
Do we really need this? Can't we just make it simple, and suggest the approach below?

`git clone https://github.com/confluentinc/cp-ansible.git ansible_collections/confluent/platform`

Reply:
good question, i assumed that would fail if the in between dirs did not exist, ill test

Reply:
works great, nice suggestion

Reply:
If someone wants to move their existing clone, they probably know how to do it. 😄